tsl/test/expected/recompression_integrity_tests.out

```diff
@@ -663,3 +663,43 @@ DROP TABLE IF EXISTS recomp_direct_compress CASCADE;
 RESET timescaledb.enable_direct_compress_insert;
 RESET timescaledb.enable_direct_compress_insert_sort_batches;
 RESET timescaledb.enable_direct_compress_insert_client_sorted;
+-- Test Case 6: Disabled in-memory recompression GUC
+CREATE TABLE recomp_guc_test(
+    time timestamptz NOT NULL,
+    device text,
+    value float
+);
+SELECT create_hypertable('recomp_guc_test','time') \gset
+ALTER TABLE recomp_guc_test SET (
+    timescaledb.compress,
+    timescaledb.compress_segmentby='device',
+    timescaledb.compress_orderby='time'
+);
+SET timescaledb.enable_in_memory_recompression = off;
+-- Insert test data
+INSERT INTO recomp_guc_test VALUES ('2000-01-01 00:00:00', 'device1', 20.5);
+-- Compress all chunks
+SELECT compress_chunk(ch) FROM show_chunks('recomp_guc_test') ch;
+              compress_chunk              
+------------------------------------------
+ _timescaledb_internal._hyper_13_26_chunk
+
+SELECT compress_chunk(ch, recompress := true) FROM show_chunks('recomp_guc_test') ch LIMIT 1;
+NOTICE:  in-memory recompression functionality disabled, set timescaledb.enable_in_memory_recompression to on
```

Comment:
I think that would be too spammy, they'd get this on every call to `compress_chunk`, right? Normally we don't do this kind of messages for GUCs disabling a feature.